apps/vr-tests-react-components/src/stories/InfoButton.stories.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import Screener from 'screener-storybook/src/screener';
+import { InfoButton } from '@fluentui/react-infobutton';
+import { storiesOf } from '@storybook/react';
+import { PopoverProps } from '@fluentui/react-popover';
+import { TestWrapperDecoratorFixedWidth } from '../utilities/TestWrapperDecorator';
+
+storiesOf('InfoButton', module)
+  .addDecorator(TestWrapperDecoratorFixedWidth)
+  .addDecorator(story => (
+    <Screener
+      steps={new Screener.Steps()
+        .snapshot('rest', { cropTo: '.testWrapper' })
+        .hover('#info-button')
+        .snapshot('hover', { cropTo: '.testWrapper' })
+        .focus('#info-button')
+        .snapshot('focus', { cropTo: '.testWrapper' })
+        .mouseDown('#info-button')
+        .snapshot('active', { cropTo: '.testWrapper' })
+        .end()}
+    >
+      {story()}
+    </Screener>
+  ))
+  .addStory(
+    'size',
+    () => (
+      <div style={{ padding: '60px' }}>
+        <InfoButton id="info-button" content="This is the content of an InfoButton." />
+      </div>
+    ),
+    {
+      includeHighContrast: true,
+      includeDarkMode: true,
+    },
+  );
+
+storiesOf('InfoButton - sizes', module)
```

Comment:
Keep this name the same so they're grouped together:
```suggestion
storiesOf('InfoButton', module)
```